scripts/local-install.sh

```diff
@@ -30,15 +40,15 @@ if [ ! -d "$2" ]; then
   exit 1
 fi
 
-rm -rf ~/.yarn/berry/cache/\@eppo-*
-
 ### Create local package via "npm pack"
 pushd "$1" > /dev/null
-rm -rf package pack.out pack.err && npm pack 2> pack.err > pack.out
-COMPRESSED_PACKAGE="$(tail -1 pack.out)"
+mkdir -p /tmp/packages
+PACKAGE_DIR=/tmp/packages/"$(basename "$(pwd)")"
+rm -rf "${PACKAGE_DIR}" /tmp/pack.out && npm pack > /tmp/pack.out
```

Comment:
piping errors to `pack.err` wasn't necessary and it suppressed errors when `npm pack` failed